packages/formatters/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@stoplight/spectral-formatters",
+  "version": "1.0.0",
+  "homepage": "https://github.com/stoplightio/spectral",
+  "bugs": "https://github.com/stoplightio/spectral/issues",
+  "author": "Stoplight <support@stoplight.io>",
+  "engines": {
+    "node": "^12.20 || >=14.13"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stoplightio/spectral.git"
+  },
+  "scripts": {
+    "release": "semantic-release -e semantic-release-monorepo",
+    "postbuild": "copyfiles -u 1 \"./src/html/*.html\" \"./dist/\""
```

Comment:
this no longer exists. I replaced it with prebuild, prelint, etc.

Reply:
Updated